[2023-08-24 15:31:01,503] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-24 15:31:01,503] logger_util.py->write_log line:61 [INFO] 接口名称：创建内部通道
[2023-08-24 15:31:01,503] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-24 15:31:01,504] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-24 15:31:01,504] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-24 15:31:01,504] logger_util.py->write_log line:61 [INFO] 请求参数：{'type': 3, 'name': '内部1710', 'description': '', 'config': None}
[2023-08-24 15:31:01,504] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-24 15:31:01,530] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}, {'contains': 'id'}]
[2023-08-24 15:31:01,530] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 129, 'parentId': None, 'sort': 8, 'name': '内部1710', 'description': '', 'type': 3, 'typeTag': None, 'config': None, 'enable': True, 'createTime': '2023-08-24T15:31:01.5236055+08:00', 'updateTime': '2023-08-24T15:31:01.5236059+08:00'}
[2023-08-24 15:31:01,530] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-24 15:31:01,530] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-24 15:31:01,535] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-24 15:31:01,535] logger_util.py->write_log line:61 [INFO] 接口名称：创建内部通道1
[2023-08-24 15:31:01,535] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-24 15:31:01,535] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-24 15:31:01,535] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-24 15:31:01,536] logger_util.py->write_log line:61 [INFO] 请求参数：{'type': 3, 'name': '内部通道1', 'description': '', 'config': None}
[2023-08-24 15:31:01,536] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-24 15:31:01,549] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 200}}, {'contains': 'id'}]
[2023-08-24 15:31:01,549] logger_util.py->write_log line:61 [INFO] 实际结果：{'id': 130, 'parentId': None, 'sort': 9, 'name': '内部通道1', 'description': '', 'type': 3, 'typeTag': None, 'config': None, 'enable': True, 'createTime': '2023-08-24T15:31:01.5433691+08:00', 'updateTime': '2023-08-24T15:31:01.5433701+08:00'}
[2023-08-24 15:31:01,550] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-24 15:31:01,550] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-24 15:31:01,555] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-24 15:31:01,555] logger_util.py->write_log line:61 [INFO] 接口名称：校验内部通道必填项-通道名为空
[2023-08-24 15:31:01,555] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-24 15:31:01,555] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-24 15:31:01,555] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-24 15:31:01,556] logger_util.py->write_log line:61 [INFO] 请求参数：{'type': 3, 'name': '', 'description': '', 'config': None}
[2023-08-24 15:31:01,556] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-24 15:31:01,562] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 400}}, {'equals': {'content': '通道名称不能为空'}}]
[2023-08-24 15:31:01,562] logger_util.py->write_log line:61 [INFO] 实际结果：通道名称不能为空
[2023-08-24 15:31:01,562] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-24 15:31:01,562] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 接口名称：校验内部通道必填项-通道名重复
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 请求参数：{'type': 3, 'name': '内部通道1', 'description': '', 'config': None}
[2023-08-24 15:31:01,568] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-24 15:31:01,574] logger_util.py->write_log line:61 [INFO] 预期结果：[{'equals': {'status_code': 400}}, {'equals': {'content': '通道名称不能重复'}}]
[2023-08-24 15:31:01,574] logger_util.py->write_log line:61 [INFO] 实际结果：通道名称不能重复
[2023-08-24 15:31:01,574] logger_util.py->write_log line:61 [INFO] 接口请求成功！
[2023-08-24 15:31:01,574] logger_util.py->write_log line:61 [INFO] 
-------------接口请求结束-----------

[2023-08-24 15:31:01,584] logger_util.py->write_log line:61 [INFO] 
-------------接口请求开始-----------
[2023-08-24 15:31:01,584] logger_util.py->write_log line:61 [INFO] 接口名称：删除通道1
[2023-08-24 15:31:01,584] logger_util.py->write_log line:61 [INFO] 接口方式：post
[2023-08-24 15:31:01,585] logger_util.py->write_log line:61 [INFO] 接口路径：http://127.0.0.1:18000/api/v1/variable/tunnel/130
[2023-08-24 15:31:01,585] logger_util.py->write_log line:61 [INFO] 请求头：{}
[2023-08-24 15:31:01,585] logger_util.py->write_log line:61 [INFO] 请求参数：{'type': 3, 'name': '内部通道1', 'description': '', 'config': None}
[2023-08-24 15:31:01,585] logger_util.py->write_log line:61 [INFO] 文件上传：None
[2023-08-24 15:31:01,587] logger_util.py->error_log line:65 [INFO] 断言失败:status_code不等于200
[2023-08-24 15:31:01,588] logger_util.py->write_log line:61 [INFO] 实际结果：
[2023-08-24 15:31:01,588] logger_util.py->write_log line:61 [INFO] 接口请求失败！
[2023-08-24 15:31:01,588] logger_util.py->write_log line:61 [INFO] -------------接口请求结束-----------

[2023-08-24 15:31:01,588] logger_util.py->error_log line:65 [INFO] 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 356, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

[2023-08-24 15:31:01,589] logger_util.py->error_log line:65 [INFO] 分析YAML文件异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 356, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 255, in analysis_yaml
    self.validate_result(yq_result, return_text, status_code)
  File "E:\Project_python\api_cms\common\requests_util.py", line 413, in validate_result
    error_log("断言异常：异常信息: %s" % str(traceback.format_exc()))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言异常：异常信息: Traceback (most recent call last):
  File "E:\Project_python\api_cms\common\requests_util.py", line 356, in validate_result
    error_log("断言失败:"+assert_key+"不等于"+str(assert_value))
  File "E:\Project_python\api_cms\common\logger_util.py", line 66, in error_log
    raise Exception(log_message)
Exception: 断言失败:status_code不等于200


